workflow/hosts.py

```diff
@@ -15,14 +15,15 @@ class Host:
     """
 
     SUPPORTED_HOSTS = ['HERA', 'ORION', 'JET',
-                       'WCOSS2', 'S4', 'CONTAINER']
+                       'WCOSS2', 'S4', 'CONTAINER', 'AWSPW']
```

Comment:
```suggestion
                       'WCOSS2', 'S4', 'CONTAINER', 'NOAACLOUD']
```

Reply:
May be there is a need to distinguish between Azure, Google and AWS later.
But for now, lets keep with the ufs-weather-model